hub-backend/src/main/java/org/panda_lang/hub/authentication/AuthenticationEntryPoint.kt

```diff
@@ -17,15 +17,12 @@ package org.panda_lang.hub.authentication
 
 import org.springframework.security.core.AuthenticationException
 import org.springframework.security.web.AuthenticationEntryPoint
-import java.io.IOException
 import javax.servlet.http.HttpServletRequest
 import javax.servlet.http.HttpServletResponse
 
 internal class AuthenticationEntryPoint : AuthenticationEntryPoint {
 
-    @Throws(IOException::class)
```

Comment:
Kotlin code does not require `@Throws` tags for anything and I believe the app should run and compile fine without them being present.

These tags are only used if third-party java code is written using the compiled kotlin jar, something which is probably not going to happen as this is a standalone project.